WooCommerce/Classes/Styles/Style.swift

```diff
@@ -88,10 +89,11 @@ class DefaultStyle: Style {
     let thinCaptionFont                 = DefaultStyle.fontForTextStyle(.caption1,
                                                                         weight: .thin,
                                                                         maximumPointSize: DefaultStyle.maxFontSize)
+    let footerLabelFont                 = UIFont.font(forStyle: .footnote, weight: .regular)
```

Comment:
If we wanted this label to support Dynamic Type we'd probably should do something like 

```
DefaultStyle.fontForTextStyle(.footnote,
                                                                        weight: .regular,
                                                                        maximumPointSize: DefaultStyle.maxFontSize)
```

In any case, it seems like the label does not update the font size. I am not sure if that's a problem, or something we can defer to another PR?
<img src="https://user-images.githubusercontent.com/2722505/61556169-315eec00-aa2f-11e9-9296-b6360479549e.png" width="350"/>

Reply:
+1 on Dynamic Type support! It's currently supported on `develop` 😄 but feel free to handle it in a separate PR